mesh_map/src/util.cpp

```diff
@@ -42,8 +42,181 @@
 #include <tf2/LinearMath/Transform.h>
 #include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
 
+#include <filesystem>
+
+namespace fs = std::filesystem;
+
 namespace mesh_map
 {
+
+// again this function
```

Comment:
What do you mean by that? Did you write / see this function someplace else already?

Reply:
This was just a joke. Because in 2024 I still have to write a split_by_delimiter function for a std::string. I removed it

Reply:
I see ^^